pages/linux/update-rc.d.md

```diff
@@ -0,0 +1,19 @@
+# update-rc.d
+
+> Install and remove service.
```

Comment:
This doesn't make grammatical sense. Try "services" instead.

Also, what init systems does this work with? Can I use it with `systemd`, or do I need `upstart`? Perhaps it's actually `openrc` that this interacts with. Including this information can avoid confusion.